packages/@aws-cdk/toolkit-lib/lib/api/notices/web-data-source.ts

```diff
@@ -44,7 +44,11 @@ export class WebsiteNoticeDataSource implements NoticeDataSource {
   }
 
   async fetch(): Promise<Notice[]> {
-    const timeout = 3000;
+    // We are observing lots of timeouts when running in a massively parallel
+    // integration test environment, so wait for a longer timeout there.
+    //
+    // In production, have a short timeout to not hold up the user experience.
+    const timeout = process.env.TESTING ? 30_000 : 3_000;
```

Comment:
Should we have a more unique env name here? This name feels generic enough for customers to have it set in their enviornment.